web/themes/custom/atrium/includes/page.inc

```diff
@@ -185,10 +186,30 @@ function _atrium_translation_menu() {
       ? str_replace("/$langcode", '', $current_path)
       : $current_path;
   }
-  // Create links for dropbutton component if translated content exists for page.
+  // Create links for dropbutton component if translated content or webform exists for page.
   $links = [];
   foreach ($languages as $langcode => $language) {
     $node = \Drupal::routeMatch()->getParameter('node');
+    $webform = \Drupal::routeMatch()->getParameter('webform');
+    
+    if ($webform instanceof Drupal\webform\Entity\Webform && $webform->hasTranslation($langcode)) {
+      // You can get webform id and anything else you need from the webform object.
+      $translated_path = \Drupal::service('path_alias.manager')->getAliasByPath('/webform/'. $webform->id(), $langcode);
+      $prefixed_translated_path = '/' . $langcode . $translated_path;
+
+      // Default language paths have no prefix.
+      $langcode = $langcode == $default_language->getId() ? '' : "/$langcode";
+      if($webform != null
+        && $prefixed_translated_path != $prefixed_current_path
+        && $prefixed_translated_path != '/en' . $prefixed_current_path){
+        $links[] = [
+          'title' => $language->getName(),
+          'href' => "$host$langcode$translated_path",
+        ];
+
+      }
```

Comment:
This whole block is similar to the node block below.  You can collapse them, and pass in the variables for lines that are different between the two.

Reply:
Created separate function for same set of code for webform and node.I had to change code little more to fulfill conditions for node and webform. Please let me know if I need to change anymore or any code refactor needs to be done. Thanks!